include/tiramisu/core.h

```diff
@@ -4021,6 +4024,50 @@ class computation
     virtual void skew(int i, int j, int k, int l, int factor);
 
 
+    /*
+      this perform a general polyhedral transformation i' = a*i +b*j under det(A)=1 constraint for j' : b!=0 & a >=0
+  
+     */
+
+    /**
+      * 
+      * 
+      * Apply a general polyhedral transformation on the loop levels \p i and \p j . 
+      *  and replaced with new var j' i' such as :
+      * i' = a*i +b*j 
+      * guarentees that the automatically computed j' creates a new iteration domain where loop increment is always =1
```

Comment:
I would replace the previous 4 lines by:

      * Apply skewing on the loop levels \p i and \p j and replace them with the new loop levels i' and j' such that:
      * i' = a*i +b*j 
      * j' is calculated automatically to guarantee that its increment is always 1.